src/browser/components/RequestsProcessor/GetBalance/GetBalance.js

```diff
@@ -5,34 +5,34 @@ import { wallet } from '@cityofzion/neon-js';
 import balanceShape from 'account/shapes/balanceShape';
 import { NEO, GAS } from 'shared/values/assets';
 
-function isValidScriptHash(scriptHash) {
-  return wallet.isScriptHash(scriptHash) || [NEO, GAS].includes(scriptHash);
+function isValidAssetHash(asset) {
+  return wallet.isScriptHash(asset) || [NEO, GAS].includes(asset);
 }
 
 export default class GetBalance extends React.Component {
   static propTypes = {
     balances: arrayOf(balanceShape).isRequired,
-    scriptHash: string.isRequired,
+    asset: string.isRequired,
```

Comment:
We'll want to support tokens in the future too.  Should we just refer to it as an "asset" always?  I'm okay with it, just want to make sure we're explicitly making that decision.

Reply:
This is consistent with `send`, which is why I changed it actually 😄